rust/Makefile

```diff
@@ -11,8 +11,17 @@ always-$(CONFIG_RUST) += exports_core_generated.h
 obj-$(CONFIG_RUST) += helpers.o
 CFLAGS_REMOVE_helpers.o = -Wmissing-prototypes -Wmissing-declarations
 
-always-$(CONFIG_RUST) += libmacros.so
-no-clean-files += libmacros.so
+UNAME := $(shell uname)
```

Comment:
No clue how to close stdin with make for the rust process. so here using uname. We could add a python script, but I don't see an advantage, just more complexity.

Reply:
You can use `echo | rustc ...` I think.

Reply:
good idea. seems to work, will try to integrate it